modules/cnf-configuring-cvl-nic-as-oc.adoc

```diff
@@ -0,0 +1,191 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+[id="cnf-configuring-cvl-nic-as-ptp-slave_{context}"]
+= Configuring linuxptp services as ordinary clock for Intel Columbiaville NIC
+
+You can configure `linuxptp` services as a single ordinary clock for nodes with an Intel 800-Series Columbiaville NIC installed.
+
+.Prerequisites
+
+* Install one or more Intel 800-Series Columbiaville NICs in your cluster bare-metal hosts.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `PtpConfig` CR for nodes with matching role.
+
+.. Save the following YAML in the `cvl-ptp-ordinary-clock.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: slave
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "slave"
```

Comment:
Probably you should add scheduling information in ptp-config
```
 profile: 
  - name: "profile1" 
    ptp4lOpts: "-2 --summary_interval -4" 
    phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"
    ptpSchedulingPolicy: "SCHED_FIFO"  
    ptpSchedulingPriority: 32

```

Reply:
Why are we not setting scheduling config this in: https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.10/ztp/source-crs/PtpConfigSlaveCvl.yaml